src/utils/bivariate/labelFormatters.test.ts

```diff
@@ -0,0 +1,149 @@
+import { expect, describe, it } from 'vitest';
```

Comment:
If you rebase this on latest main we can have a look how good is the coverage now :)